Fixing more inconsistencies

djdanielsson opened the pull request:
<!-- markdownlint-disable -->
# What does this PR do?
<!--- Brief explanation of the code or documentation change you've made -->
AI found some more inconsistencies

# How should this be tested?
<!--- Automated tests are preferred, but not always doable - especially for infrastructure. Include commands to run your new feature, and also post-run commands to validate that it worked. (please use code blocks to format code samples) -->
manually, which I have not so needs thorough testing to ensure arg specs aren't messed up. I think it just uncommented them all but they were created a while ago so needs testing.

# Is there a relevant Issue open for this?
<!--- Provide a link to any open issues that describe the problem you are solving. -->
no

# Other Relevant info, PRs, etc
<!--- Please provide link to other PRs that may be related (blocking, resolves, etc. etc.) -->
none

djdanielsson committed:
new file:   ALIAS_SUPPORT.md
	modified:   roles/collect_async_status/defaults/main.yml
	modified:   roles/controller_ad_hoc_command/meta/argument_specs.yml
	modified:   roles/controller_ad_hoc_command_cancel/meta/argument_specs.yml
	modified:   roles/controller_applications/defaults/main.yml
	modified:   roles/controller_applications/meta/argument_specs.yml
	modified:   roles/controller_bulk_host_create/defaults/main.yml
	modified:   roles/controller_bulk_job_launch/defaults/main.yml
	modified:   roles/controller_credential_input_sources/defaults/main.yml
	modified:   roles/controller_credential_input_sources/meta/argument_specs.yml
	modified:   roles/controller_credential_types/defaults/main.yml
	modified:   roles/controller_credential_types/meta/argument_specs.yml
	modified:   roles/controller_credentials/defaults/main.yml
	modified:   roles/controller_credentials/meta/argument_specs.yml
	modified:   roles/controller_execution_environments/defaults/main.yml
	modified:   roles/controller_execution_environments/meta/argument_specs.yml
	modified:   roles/controller_host_groups/defaults/main.yml
	modified:   roles/controller_host_groups/meta/argument_specs.yml
	modified:   roles/controller_hosts/defaults/main.yml
	modified:   roles/controller_hosts/meta/argument_specs.yml
	modified:   roles/controller_instance_groups/defaults/main.yml
	modified:   roles/controller_instance_groups/meta/argument_specs.yml
	modified:   roles/controller_instances/defaults/main.yml
	modified:   roles/controller_instances/meta/argument_specs.yml
	modified:   roles/controller_inventories/defaults/main.yml
	modified:   roles/controller_inventories/meta/argument_specs.yml
	modified:   roles/controller_inventory_source_update/defaults/main.yml
	modified:   roles/controller_inventory_source_update/meta/argument_specs.yml
	modified:   roles/controller_inventory_sources/defaults/main.yml
	modified:   roles/controller_inventory_sources/meta/argument_specs.yml
	modified:   roles/controller_job_launch/meta/argument_specs.yml
	modified:   roles/controller_job_templates/defaults/main.yml
	modified:   roles/controller_job_templates/meta/argument_specs.yml
	modified:   roles/controller_jobs_cancel/meta/argument_specs.yml
	modified:   roles/controller_labels/defaults/main.yml
	modified:   roles/controller_labels/meta/argument_specs.yml
	modified:   roles/controller_license/meta/argument_specs.yml
	modified:   roles/controller_notification_templates/defaults/main.yml
	modified:   roles/controller_notification_templates/meta/argument_specs.yml
	modified:   roles/controller_organizations/defaults/main.yml
	modified:   roles/controller_organizations/meta/argument_specs.yml
	modified:   roles/controller_project_update/meta/argument_specs.yml
	modified:   roles/controller_projects/defaults/main.yml
	modified:   roles/controller_projects/meta/argument_specs.yml
	modified:   roles/controller_roles/defaults/main.yml
	modified:   roles/controller_roles/meta/argument_specs.yml
	modified:   roles/controller_schedules/defaults/main.yml
	modified:   roles/controller_schedules/meta/argument_specs.yml
	modified:   roles/controller_settings/defaults/main.yml
	modified:   roles/controller_settings/meta/argument_specs.yml
	modified:   roles/controller_teams/defaults/main.yml
	modified:   roles/controller_teams/meta/argument_specs.yml
	modified:   roles/controller_users/defaults/main.yml
	modified:   roles/controller_users/meta/argument_specs.yml
	modified:   roles/controller_workflow_job_templates/defaults/main.yml
	modified:   roles/controller_workflow_job_templates/meta/argument_specs.yml
	modified:   roles/controller_workflow_launch/meta/argument_specs.yml
	modified:   roles/eda_users/meta/argument_specs.yml
	modified:   roles/gateway_applications/defaults/main.yml
	modified:   roles/gateway_applications/meta/argument_specs.yml
	modified:   roles/gateway_authenticator_maps/defaults/main.yml
	modified:   roles/gateway_authenticator_maps/meta/argument_specs.yml
	modified:   roles/gateway_authenticators/defaults/main.yml
	modified:   roles/gateway_authenticators/meta/argument_specs.yml
	modified:   roles/gateway_http_ports/defaults/main.yml
	modified:   roles/gateway_http_ports/meta/argument_specs.yml
	modified:   roles/gateway_organizations/defaults/main.yml
	modified:   roles/gateway_organizations/meta/argument_specs.yml
	modified:   roles/gateway_role_definitions/README.md
	modified:   roles/gateway_role_definitions/defaults/main.yml
	modified:   roles/gateway_role_definitions/meta/argument_specs.yml
	modified:   roles/gateway_role_team_assignments/defaults/main.yml
	modified:   roles/gateway_role_team_assignments/meta/argument_specs.yml
	modified:   roles/gateway_role_user_assignments/defaults/main.yml
	modified:   roles/gateway_role_user_assignments/meta/argument_specs.yml
	modified:   roles/gateway_routes/defaults/main.yml
	modified:   roles/gateway_routes/meta/argument_specs.yml
	modified:   roles/gateway_service_clusters/defaults/main.yml
	modified:   roles/gateway_service_clusters/meta/argument_specs.yml
	modified:   roles/gateway_service_keys/defaults/main.yml
	modified:   roles/gateway_service_keys/meta/argument_specs.yml
	modified:   roles/gateway_service_nodes/defaults/main.yml
	modified:   roles/gateway_service_nodes/meta/argument_specs.yml
	modified:   roles/gateway_services/defaults/main.yml
	modified:   roles/gateway_services/meta/argument_specs.yml
	modified:   roles/gateway_settings/defaults/main.yml
	modified:   roles/gateway_settings/meta/argument_specs.yml
	modified:   roles/gateway_teams/defaults/main.yml
	modified:   roles/gateway_teams/meta/argument_specs.yml
	modified:   roles/gateway_users/defaults/main.yml
	modified:   roles/gateway_users/meta/argument_specs.yml
	modified:   roles/hub_publish/meta/argument_specs.yml

djdanielsson committed:
Merge branch 'redhat-cop:devel' into fixing_more_inconsistencies

akira6592 commented:
@djdanielsson Thank you for fixing!

If we change to the default values of `*_async_retries` `30` to `50`, I think we should apply to the `README.md`.

djdanielsson committed:
modified:   roles/collect_async_status/README.md
	modified:   roles/controller_applications/README.md
	modified:   roles/controller_bulk_host_create/README.md
	modified:   roles/controller_bulk_job_launch/README.md
	modified:   roles/controller_credential_input_sources/README.md
	modified:   roles/controller_credential_types/README.md
	modified:   roles/controller_credentials/README.md
	modified:   roles/controller_execution_environments/README.md
	modified:   roles/controller_host_groups/README.md
	modified:   roles/controller_hosts/README.md
	modified:   roles/controller_instance_groups/README.md
	modified:   roles/controller_instances/README.md
	modified:   roles/controller_inventories/README.md
	modified:   roles/controller_inventory_source_update/README.md
	modified:   roles/controller_inventory_sources/README.md
	modified:   roles/controller_job_templates/README.md
	modified:   roles/controller_labels/README.md
	modified:   roles/controller_notification_templates/README.md
	modified:   roles/controller_organizations/README.md
	modified:   roles/controller_projects/README.md
	modified:   roles/controller_roles/README.md
	modified:   roles/controller_schedules/README.md
	modified:   roles/controller_settings/README.md
	modified:   roles/controller_teams/README.md
	modified:   roles/controller_users/README.md
	modified:   roles/controller_workflow_job_templates/README.md
	modified:   roles/gateway_role_definitions/README.md
	modified:   roles/gateway_role_definitions/tasks/main.yml
	modified:   roles/gateway_role_team_assignments/README.md
	modified:   roles/gateway_role_team_assignments/tasks/main.yml
	modified:   roles/gateway_role_user_assignments/README.md
	modified:   tests/templated_role_example/README.md

djdanielsson committed:
Merge branch 'fixing_more_inconsistencies' of github.com:djdanielsson/infra.aap_configuration into fixing_more_inconsistencies

djdanielsson committed:
deleted:    ALIAS_SUPPORT.md
	modified:   roles/controller_applications/meta/argument_specs.yml

djdanielsson committed:
modified:   roles/controller_applications/meta/argument_specs.yml
	modified:   roles/controller_bulk_host_create/meta/argument_specs.yml
	modified:   roles/controller_credential_input_sources/meta/argument_specs.yml
	modified:   roles/controller_credential_types/meta/argument_specs.yml
	modified:   roles/controller_credentials/meta/argument_specs.yml
	modified:   roles/controller_execution_environments/meta/argument_specs.yml
	modified:   roles/controller_host_groups/meta/argument_specs.yml
	modified:   roles/controller_hosts/meta/argument_specs.yml
	modified:   roles/controller_instance_groups/meta/argument_specs.yml
	modified:   roles/controller_instances/meta/argument_specs.yml
	modified:   roles/controller_inventories/meta/argument_specs.yml
	modified:   roles/controller_inventory_source_update/meta/argument_specs.yml
	modified:   roles/controller_inventory_sources/meta/argument_specs.yml
	modified:   roles/controller_job_templates/meta/argument_specs.yml
	modified:   roles/controller_labels/meta/argument_specs.yml
	modified:   roles/controller_notification_templates/meta/argument_specs.yml
	modified:   roles/controller_organizations/meta/argument_specs.yml
	modified:   roles/controller_project_update/meta/argument_specs.yml
	modified:   roles/controller_projects/meta/argument_specs.yml
	modified:   roles/controller_roles/meta/argument_specs.yml
	modified:   roles/controller_schedules/meta/argument_specs.yml
	modified:   roles/controller_settings/meta/argument_specs.yml
	modified:   roles/controller_teams/meta/argument_specs.yml
	modified:   roles/controller_users/meta/argument_specs.yml
	modified:   roles/controller_workflow_job_templates/meta/argument_specs.yml
	modified:   roles/controller_workflow_launch/meta/argument_specs.yml
	modified:   roles/dispatch/meta/argument_specs.yml
	modified:   roles/eda_controller_tokens/meta/argument_specs.yml
	modified:   roles/eda_credential_types/meta/argument_specs.yml
	modified:   roles/eda_credentials/meta/argument_specs.yml
	modified:   roles/eda_decision_environments/meta/argument_specs.yml
	modified:   roles/eda_event_streams/meta/argument_specs.yml
	modified:   roles/eda_projects/meta/argument_specs.yml
	modified:   roles/eda_rulebook_activations/meta/argument_specs.yml
	modified:   roles/eda_users/meta/argument_specs.yml
	modified:   roles/gateway_applications/meta/argument_specs.yml
	modified:   roles/gateway_authenticator_maps/meta/argument_specs.yml
	modified:   roles/gateway_authenticators/meta/argument_specs.yml
	modified:   roles/gateway_http_ports/meta/argument_specs.yml
	modified:   roles/gateway_organizations/meta/argument_specs.yml
	modified:   roles/gateway_role_definitions/meta/argument_specs.yml
	modified:   roles/gateway_role_team_assignments/meta/argument_specs.yml
	modified:   roles/gateway_role_user_assignments/meta/argument_specs.yml
	modified:   roles/gateway_routes/meta/argument_specs.yml
	modified:   roles/gateway_service_clusters/meta/argument_specs.yml
	modified:   roles/gateway_service_keys/meta/argument_specs.yml
	modified:   roles/gateway_service_nodes/meta/argument_specs.yml
	modified:   roles/gateway_services/meta/argument_specs.yml
	modified:   roles/gateway_settings/meta/argument_specs.yml
	modified:   roles/gateway_teams/meta/argument_specs.yml
	modified:   roles/gateway_users/meta/argument_specs.yml
	modified:   roles/hub_collection/meta/argument_specs.yml
	modified:   roles/hub_collection_remote/meta/argument_specs.yml
	modified:   roles/hub_collection_repository/meta/argument_specs.yml
	modified:   roles/hub_collection_repository_sync/meta/argument_specs.yml
	modified:   roles/hub_ee_image/meta/argument_specs.yml
	modified:   roles/hub_ee_registry/meta/argument_specs.yml
	modified:   roles/hub_ee_registry_index/meta/argument_specs.yml
	modified:   roles/hub_ee_registry_sync/meta/argument_specs.yml
	modified:   roles/hub_ee_repository/meta/argument_specs.yml
	modified:   roles/hub_ee_repository_sync/meta/argument_specs.yml
	modified:   roles/hub_group/meta/argument_specs.yml
	modified:   roles/hub_group_roles/meta/argument_specs.yml
	modified:   roles/hub_namespace/meta/argument_specs.yml
	modified:   roles/hub_publish/meta/argument_specs.yml
	modified:   roles/hub_role/meta/argument_specs.yml
	modified:   roles/hub_user/meta/argument_specs.yml

sean-m-sullivan reviewed (approved):
LGTM If it passes an integration test would be the only thing I'd check on as the argument spec changes.

djdanielsson committed:
new file:   changelogs/fragments/updated_async_retries.yml

Tompage1994 commented:
We decided NOT to include the sub-options of each variable as it could potentially break some people's automation if they choose to hold additional fields under a variable (which there are some use cases for). In this case we would break the role for them. I propose keeping them commented out

djdanielsson commented:
I am conflicted on this... I would really like to have it there because it can really help catch issues but I understand the concern... I wonder how many people are passing variables that are not direct arguments.

djdanielsson committed:
modified:   roles/controller_ad_hoc_command/meta/argument_specs.yml
	modified:   roles/controller_ad_hoc_command_cancel/meta/argument_specs.yml
	modified:   roles/controller_applications/meta/argument_specs.yml
	modified:   roles/controller_credential_input_sources/meta/argument_specs.yml
	modified:   roles/controller_credential_types/meta/argument_specs.yml
	modified:   roles/controller_credentials/meta/argument_specs.yml
	modified:   roles/controller_execution_environments/meta/argument_specs.yml
	modified:   roles/controller_host_groups/meta/argument_specs.yml
	modified:   roles/controller_hosts/meta/argument_specs.yml
	modified:   roles/controller_instance_groups/meta/argument_specs.yml
	modified:   roles/controller_instances/meta/argument_specs.yml
	modified:   roles/controller_inventories/meta/argument_specs.yml
	modified:   roles/controller_inventory_source_update/meta/argument_specs.yml
	modified:   roles/controller_inventory_sources/meta/argument_specs.yml
	modified:   roles/controller_job_launch/meta/argument_specs.yml
	modified:   roles/controller_job_templates/meta/argument_specs.yml
	modified:   roles/controller_jobs_cancel/meta/argument_specs.yml
	modified:   roles/controller_labels/meta/argument_specs.yml
	modified:   roles/controller_license/meta/argument_specs.yml
	modified:   roles/controller_notification_templates/meta/argument_specs.yml
	modified:   roles/controller_organizations/meta/argument_specs.yml
	modified:   roles/controller_project_update/meta/argument_specs.yml
	modified:   roles/controller_projects/meta/argument_specs.yml
	modified:   roles/controller_roles/meta/argument_specs.yml
	modified:   roles/controller_schedules/meta/argument_specs.yml
	modified:   roles/controller_settings/meta/argument_specs.yml
	modified:   roles/controller_teams/meta/argument_specs.yml
	modified:   roles/controller_users/meta/argument_specs.yml
	modified:   roles/controller_workflow_job_templates/meta/argument_specs.yml
	modified:   roles/controller_workflow_launch/meta/argument_specs.yml
	modified:   roles/gateway_applications/meta/argument_specs.yml
	modified:   roles/gateway_authenticator_maps/meta/argument_specs.yml
	modified:   roles/gateway_authenticators/meta/argument_specs.yml
	modified:   roles/gateway_http_ports/meta/argument_specs.yml
	modified:   roles/gateway_organizations/meta/argument_specs.yml
	modified:   roles/gateway_role_definitions/meta/argument_specs.yml
	modified:   roles/gateway_role_team_assignments/meta/argument_specs.yml
	modified:   roles/gateway_role_user_assignments/meta/argument_specs.yml
	modified:   roles/gateway_routes/meta/argument_specs.yml
	modified:   roles/gateway_service_clusters/meta/argument_specs.yml
	modified:   roles/gateway_service_keys/meta/argument_specs.yml
	modified:   roles/gateway_service_nodes/meta/argument_specs.yml
	modified:   roles/gateway_services/meta/argument_specs.yml
	modified:   roles/gateway_teams/meta/argument_specs.yml

djdanielsson committed:
Merge branch 'devel' into fixing_more_inconsistencies
